components/backup-stream/src/router.rs

```diff
@@ -544,26 +545,44 @@ impl TempFileKey {
         }
     }
 
-    fn path_to_log_file(&self, min_ts: u64) -> String {
+    fn format_date_time(ts: u64) -> impl Display {
+        use chrono::prelude::*;
+        let millis = TimeStamp::physical(ts.into());
+        let dt = Utc.timestamp_millis(millis as _);
+        fail::fail_point!("stream_format_date_time", |s| {
+            return dt.format(&s.unwrap_or("%Y%m".to_owned())).to_string();
+        });
+        #[cfg(feature = "failpoints")]
+        return dt.format("%Y%m%d").to_string();
+        #[cfg(not(feature = "failpoints"))]
+        return dt.format("%Y%m%d");
+    }
+
+    fn path_to_log_file(&self, min_ts: u64, max_ts: u64) -> String {
         format!(
-            // "/v1/t{:012}/{:012}-{}.log",
-            "v1_t{:012}_{:012}-{}.log",
+            "t{:08}/{}-{:012}-{}.log",
             self.table_id,
+            // We may delete a range of files, so using the max_ts for preventing remove some records wrong.
+            Self::format_date_time(max_ts),
             min_ts,
             uuid::Uuid::new_v4()
         )
     }
 
-    fn path_to_schema_file(min_ts: u64) -> String {
-        // format!("/v1/m/{:012}-{}.log", min_ts, uuid::Uuid::new_v4())
-        format!("v1_m{:012}-{}.log", min_ts, uuid::Uuid::new_v4())
+    fn path_to_schema_file(min_ts: u64, max_ts: u64) -> String {
+        format!(
+            "schema-meta/{}-{:012}-{}.log",
+            Self::format_date_time(max_ts),
+            min_ts,
+            uuid::Uuid::new_v4(),
+        )
     }
 
-    fn file_name(&self, min_ts: TimeStamp) -> String {
+    fn file_name(&self, min_ts: TimeStamp, max_ts: TimeStamp) -> String {
```

Comment:
Shall we make a pr about relative code in `br` ?

Reply:
Local storage in BR supports subdirectories. And the path of log files are defined in the meta files(which's location is unchanged), so it's probably no changes needing for BR.